salmon/tagger/combine.py

```diff
@@ -177,9 +191,13 @@ def combine_tracks(base, meta):
             if not btrack["replay_gain"]:
                 btrack["replay_gain"] = track["replay_gain"]
                 btrack["title"] = track["title"]
-            if track["tracktotal"] and track["disctotal"]:
+            if not btrack["tracktotal"]:
```

Comment:
```suggestion
            if not btrack["tracktotal"] and track["tracktotal"]:
```
Does this still need a None check before the assignment?